src/Server/MySQLHandlerFactory.h

```diff
@@ -23,14 +23,7 @@ class MySQLHandlerFactory : public TCPServerConnectionFactory
     LoggerPtr log;
 
 #if USE_SSL
-    struct RSADeleter
-    {
-        void operator()(RSA * ptr) { RSA_free(ptr); }
-    };
-    using RSAPtr = std::unique_ptr<RSA, RSADeleter>;
-
-    RSAPtr public_key;
-    RSAPtr private_key;
+    KeyPair private_key;
```

Comment:
So if "KeyPair" is a private + public key, can we name the variable then just "key"?